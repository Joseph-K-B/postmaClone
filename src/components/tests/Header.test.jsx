import React from 'react';
import { render } from '@testing-library/react';
import Header from '../app/Header';

describe('renders header with h1', () => {
    it('displays header and styling upon component mounting', () => {
        render(
            <Header />
        );
        expect(Header).toMatchSnapshot();
    });
});