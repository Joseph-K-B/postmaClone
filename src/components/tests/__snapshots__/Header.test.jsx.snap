// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header with h1 displays header and styling upon component mounting 1`] = `[Function]`;
